index.js

```diff
@@ -1,66 +1,103 @@
-// const awaitInvoke = require('./lib/rules/await-invoke');
-// const awaitPublish = require('./lib/rules/await-publish');
-const limitBeforeFirst = require('./lib/rules/limit-before-first');
-const metaDescription = require('./lib/rules/meta-description');
-const metaExport = require('./lib/rules/meta-export');
-const metaPermissions = require('./lib/rules/meta-permissions');
+// const metaPermissions = require('./lib/rules/meta-permissions');
 
 module.exports = {
   configs: {
     recommended: {
       extends: [
         'airbnb-base',
       ],
-      plugins: ['cubyn'],
       env: {
         es6: true,
-        jest: true,
         node: true,
       },
       rules: {
-        // 'cubyn/await-invoke': 'off',
-        // 'cubyn/await-publish': 'off',
-        'cubyn/limit-before-first': 'off',
-        'cubyn/meta-description': 'off',
-        'cubyn/meta-export': 'off',
-        'cubyn/meta-permissions': 'off',
+        //
+        // Cubyn rules
+        //
 
-        'max-depth': ['error', 3],
-        'max-len': ['error', 100, 2, { ignoreStrings: false }],
-        'no-param-reassign': 'off',
-        'no-plusplus': 'off',
-        'no-use-before-define': 'off',
-        complexity: ['error', 5],
+        // 'cubyn/meta-permissions': 'off',
+
+        //
+        // Airbnb rules
+        //
+
+        // Activations/updates
+
+        'func-style': ['error', 'declaration', { allowArrowFunctions: true }],
+        'id-length': ['error', { min: 3, properties: 'never' }],
```

Comment:
setting a max would be nice as well

`expectedGridDurationMinValue` for instance is hard on my eyes. this usually denotes SRP violation. i.E:

```
function checkGridDuration(value) {
  const minValue = ...; // <-- the context enables shorter var names
}
```

Reply:
hard to define what is a "long" variable, sometime I prefer big variable name to explain an expression.
Exemple pourri ;
```
const isParcelUnvalidForAfgathisthan = parcel.countr === 'an' && parcel.itemsJson[0] === 'alcool';

if (isParcelUnvalidForAfgathisthan) {
```